unit_tests/MultirankUtils.h

```diff
@@ -1,132 +0,0 @@
-//
-// Utility class for rank independent unit testing
```

Comment:
Is this deleted on purpose?

Reply:
Yes. It's been superseded by `TestUtils.h`. The more generic name reflects that it also covers testing with different precisions and execution spaces.

Testing with different dimensionalities is now covered by GTest parameterization instead of metaloops.